complyscribe/tasks/sync_cac_content_task.py

```diff
@@ -328,6 +328,7 @@ def _add_set_parameters(
         params = []
         for rule in rules:
             params.extend(rule._parameters)
+        params.extend(rules[0]._parameters)
```

Comment:
I think we should comment two lines above this line, because this will let `params` variable contains duplicate parameters. WDYT. The reason why no duplicate data in result is `param_selections` is a dictionary, it deduplicate the items.
```python
        # for rule in rules:
        #     params.extend(rule._parameters)
```

Reply:
Thanks. Updated.